ionic2/app/services/config.service.ts

```diff
@@ -25,4 +24,5 @@ export class ConfigService {
     }
   }
 
+  buildEnvironment = env.BUILD_ENV;
```

Comment:
I agree. Even better: a method `isDevelopEnvironment()`
```
get isDevelopEnvironment(): boolean {
    return env.BUILD_ENV === 'develop';
}
```
But that's just my personal preference.